eagle-topology-check/eagle-topology-app/src/main/resources/META-INF/providers/org.apache.eagle.topology.TopologyCheckAppProvider.xml

```diff
@@ -89,28 +97,36 @@
             <description>Hbase Master Principal</description>
             <value>hadoop/_HOST@EXAMPLE.COM</value>
         </property>
-
+        
+        <property>
+            <name>dataSourceConfig.hbase.kerberos.eagle.principal</name>
+            <displayName>hbaseEaglePrincipal</displayName>
+            <description>Hbase eagle Principal</description>
+            <value></value>
+        </property>
```

Comment:
Please remove this config to customize config. You can take mr history feeder as an example.